Dockerfile

```diff
@@ -9,26 +9,44 @@ RUN gem install bundler
 FROM bundler as dependencies
 
 # add nodejs, yarn, and other dependencies
-RUN curl -sL https://deb.nodesource.com/setup_9.x | bash - && \
-  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-  echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-  apt-get update && apt-get upgrade -y && \
-  apt-get install --no-install-recommends -y ca-certificates nodejs yarn \
-  build-essential libpq-dev libreoffice imagemagick graphicsmagick unzip \
-  zip ghostscript vim tesseract-ocr poppler-utils libopenjp2-tools \
-  ffmpeg qt5-default libqt5webkit5-dev xvfb xauth openjdk-11-jre \
-  --fix-missing --allow-unauthenticated
+#RUN curl -sL https://deb.nodesource.com/setup_9.x | bash - && \
+#  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+#  echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+#  apt-get update && apt-get upgrade -y && \
+#  apt-get install --no-install-recommends -y ca-certificates nodejs yarn \
+#  build-essential libpq-dev libreoffice imagemagick graphicsmagick unzip \
+#  zip ghostscript vim tesseract-ocr poppler-utils libopenjp2-tools \
+#  ffmpeg qt5-default libqt5webkit5-dev xvfb xauth openjdk-11-jre \
+#  --fix-missing --allow-unauthenticated
+
+# The alpine way
+RUN apk --no-cache update && apk --no-cache upgrade && \
+  apk add --no-cache alpine-sdk nodejs imagemagick unzip ghostscript vim yarn \
+  git sqlite sqlite-dev postgresql-dev libressl libressl-dev \
+  curl libc6-compat build-base tzdata zip autoconf automake libtool texinfo
```

Comment:
No java? I'm pretty sure fits requires `java 1.7` for `fits 1.0.5` and `java 1.8` for `fits 1.5.0`
Have you tested to see if fits runs?

Reply:
added the java-common package, and openjdk11-jre-headless to match what we were using before